src/mqt/predictor/rl/PredictorEnv.py

```diff
@@ -83,6 +85,7 @@ def __init__(
     def step(self, action: int) -> tuple[dict[str, Any], float, bool, bool, dict[Any, Any]]:
         """Executes the given action and returns the new state, the reward, whether the episode is done, whether the episode is truncated and additional information."""
         self.used_actions.append(str(self.action_set[action].get("name")))
+        print(self.action_set[action].get("name"), self.layout)
```

Comment:
This seems like a leftover from debugging. Should this be removed?

Reply:
Yes, that was accidentally. I removed it.